crates/router/src/types/api/webhooks.rs

```diff
@@ -134,7 +135,7 @@ pub trait IncomingWebhook: ConnectorCommon + Sync {
         &self,
         _request: &IncomingWebhookRequestDetails<'_>,
         _merchant_id: &str,
-        _secret: &[u8],
+        _connector_webhook_secrets: &api_models::webhooks::ConnectorWebhookSecrets,
```

Comment:
you can always access `get_webhook_source_verification_merchant_secret` in `get_webhook_source_verification_message`